projects/dashboards/src/persistence/store/dashboard-remote-store.ts

```diff
@@ -0,0 +1,64 @@
+/* tslint:disable:no-console */
+import { Injectable } from '@angular/core';
+import { EMPTY, Observable, throwError } from 'rxjs';
+import { catchError, map } from 'rxjs/operators';
+// TODO: fix import
+/* tslint:disable:import-blacklist */
+import { CustomDashboardService } from '../../../../observability/src/pages/custom-dashboards/custom-dashboard.service';
```

Comment:
I think this import needs to be done differently, given it's from a different project scope.

Reply:
Actually, I tried but the `@hypertrace/observability` module is not included in the `tsconfig.lib.json` of the `dashboards` module. When I tried to include it there was a cyclic dependency and the project would not run so had to import it like this. Any alternative approach you can suggest?
<img width="937" alt="Screenshot 2022-05-26 at 2 08 27 PM" src="https://user-images.githubusercontent.com/101543426/170451746-7fe74083-ca9e-4578-b48b-b99530427221.png">